docs/hoon/library/1b.md

```diff
@@ -135,12 +252,42 @@ Source
 Examples
 --------
 
-    >  (peg 4 1)
+    > (peg 4 1)
+    4
+
+    > (peg 1 4)
     4
-    >  (peg 4 2)
+
+    > (peg 4 2)
     8
-    >  (peg 8 45)
+
+    > (peg 4 8)
+    32
+
+    > (peg 4 (4 2))
+    32
+
+    > (peg 8 45)
+    269
+
+    > (peg 5 'a')                              
+    353
+    > `@`'a'
+    97
+
+    > (peg 0x8 0x2d)                            
     269
+    > `@`0x8
+    8
+    > `@`0x2d
+    45
+
+    > (cap (peg 4 2))                            ::`(cap a)` computes whether address `a` is in the head or the tail
+    %2
+
+    > (peg 0 4)
+    ! exit
+
 
```

Comment:
Again, some of these examples seem a little off the wall.  What's the intention here?